src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -653,7 +653,7 @@ internal unsafe struct MethodTableAuxiliaryData
         [FieldOffset(0)]
         private uint Flags;
 
-        private const uint enum_flag_CanCompareBitsOrUseFastGetHashCode = 0x0001;     // Is any field type or sub field type overrode Equals or GetHashCode
+        private const uint enum_flag_CanCompareBitsOrUseFastGetHashCode = 0x0400;     // Is any field type or sub field type overrode Equals or GetHashCode
```

Comment:
```suggestion
        private const uint enum_flag_CanCompareBitsOrUseFastGetHashCode = 0x0004;     // Is any field type or sub field type overrode Equals or GetHashCode
```
Also move it after enum_flag_HasCheckedCanCompareBitsOrUseFastGetHashCode